src/persistence/sql/cms_storage_sql_builder.e

```diff
@@ -40,6 +40,10 @@ feature -- Initialization
 
 				--| Node			
 				-- FIXME: move that initialization to node module
+				-- TODO: should we move the initialization to an
```

Comment:
External config file, or similar, why not.
I think we could provide 2 app, the frontend, and the backend.
And we would do those initialization only in the backend/admin site.

Otherwise, yes one admin is enough.
If a site needs more, it can create a "administrator" role, and manage it this way.

We need to work more on the user management side .. i.e create user from the web interface, and so on.
the rest my password or similar as well.